conjur/logic/credential_provider/credential_store_factory.py

```diff
@@ -31,7 +39,17 @@ def create_credential_store(cls):
             if KeystoreAdapter.is_keyring_accessible():
                 # pylint: disable=line-too-long
                 return KeystoreCredentialsProvider(), f'{KeystoreAdapter.get_keyring_name()} credential store'
-            # If the keystore is not unlocked we will resort to netrc
-            return FileCredentialsProvider(), DEFAULT_NETRC_FILE
```

Comment:
why was this removed? dont we need to this to return filecredentialprovider if not accessible?